public/novnc/vnc.html

```diff
@@ -15,53 +15,35 @@
     -->
     <title>noVNC</title>
 
-    <meta charset="utf-8">
```

Comment:
why have you removed all the icons and also the meta header? isnt it going to stop utf-8 stuff working?

Reply:
ok mine also removes the utf-8 for some reason?
but why have you removed all the icons?
these are needed to show pretty icons on menus etc